src/main/java/com/adobe/cq/commerce/graphql/client/impl/GraphqlClientImpl.java

```diff
@@ -71,6 +64,9 @@
     private boolean acceptSelfSignedCertificates;
     private int maxHttpConnections;
     private HttpMethod httpMethod;
+    private int conn_timeout;
```

Comment:
Please use correct camelcase names for Java variables (I'd also use the same names in the OSGi config, but you can also keep the `_` separator in the OSGi config if you want to, because this is a special allowed syntax in that particular case).

Reply:
done